Makefile

```diff
@@ -493,7 +493,7 @@ proto-update-deps:
 ### Docs
 ##############################
 update-swagger-docs: statik
-	./scripts/protoc-swagger-gen.sh
+	make proto-swagger-gen
```

Comment:
I'm not too well versed in Makefile syntax, but I think this is better handled by putting the other action to do on the main action line.
```
update-swagger-docs: statik proto-swagger-gen
```

Reply:
Indeed you chain targets instead of nesting contexts like this... refer to examples such as `proto-all`

Reply:
👍 updated @dwedul-figure @iramiller